packages/ui/src/utils/sign-in-experience.ts

```diff
@@ -27,21 +27,16 @@ const getSecondarySignInMethod = (signInMethods: SignInMethods) =>
     return methods;
   }, []);
 
-const getSignInExperienceSettings = async () => {
-  try {
-    const result = await getSignInExperience();
-    const settings = {
-      branding: result.branding,
-      languageInfo: result.languageInfo,
-      termsOfUse: result.termsOfUse,
-      primarySignInMethod: getPrimarySignInMethod(result.signInMethods),
-      secondarySignInMethods: getSecondarySignInMethod(result.signInMethods),
-    };
-
-    return { settings };
-  } catch (error: unknown) {
-    return { error };
-  }
+const getSignInExperienceSettings = async (): Promise<SignInExperienceSettings> => {
+  const result = await getSignInExperience();
+
+  return {
+    branding: result.branding,
+    languageInfo: result.languageInfo,
+    termsOfUse: result.termsOfUse,
+    primarySignInMethod: getPrimarySignInMethod(result.signInMethods),
+    secondarySignInMethods: getSecondarySignInMethod(result.signInMethods),
```

Comment:
nit: deconstruct variables from result to avoid writing `result.` numerous times

Reply:
updated